flexslider_fields/flexslider_fields.module

```diff
@@ -229,6 +233,9 @@ function flexslider_fields_field_formatter_view($entity_type, $entity, $field, $
       elseif ($display['settings']['caption'] === 'alt') {
         $item['caption'] = filter_xss($item['item']['alt']);
       }
+      elseif ($display['settings']['caption'] === 'image_field_caption' && module_exists('image_field_caption') && isset($item['item']['image_field_caption']['value'])) {
```

Comment:
I'd prefer if these were in the order of checking for the module first (so no other checks would proceed if it did not exist) but since this is the change that went into Drupal, we can start here :)